python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -75,44 +75,42 @@ class TokenizerManager:
 
     def __init__(
         self,
-        server_args: ServerArgs,
-        port_args: PortArgs,
-        model_override_args: dict = None,
+        engine_args: EngineArgs,
     ):
-        self.server_args = server_args
+        self.engine_args = engine_args
 
         # Init inter-process communication
         context = zmq.asyncio.Context(2)
         self.recv_from_detokenizer = context.socket(zmq.PULL)
-        self.recv_from_detokenizer.bind(f"tcp://127.0.0.1:{port_args.tokenizer_port}")
+        self.recv_from_detokenizer.bind(f"tcp://127.0.0.1:{engine_args.tokenizer_port}")
 
         self.send_to_router = context.socket(zmq.PUSH)
-        self.send_to_router.connect(f"tcp://127.0.0.1:{port_args.controller_port}")
+        self.send_to_router.connect(f"tcp://127.0.0.1:{engine_args.controller_port}")
 
         # Read model args
-        self.model_path = server_args.model_path
-        self.served_model_name = server_args.served_model_name
+        self.model_path = engine_args.model_path
+        self.served_model_name = engine_args.served_model_name
         self.hf_config = get_config(
             self.model_path,
-            trust_remote_code=server_args.trust_remote_code,
-            model_override_args=model_override_args,
+            trust_remote_code=engine_args.trust_remote_code,
+            model_overide_args=engine_args.model_override_args,
```

Comment:
same here; this should be `model_override_args` to match fn signature in `get_config`